openedx/features/offline_mode/utils.py

```diff
@@ -36,6 +37,8 @@ def generate_offline_content(xblock, html_data):
     try:
         save_xblock_html(tmp_dir, xblock, html_data)
         create_zip_file(tmp_dir, base_path, f'{xblock.location.block_id}.zip')
+    except Http404:
+        log.error(f'Block {xblock.location.block_id} for course {xblock.location.course_key} not found.')
     finally:
         shutil.rmtree(tmp_dir, ignore_errors=True)
```

Comment:
If there's return on line 31, it's better to make celery task return result in all outcomes, the best option is to return True/False depending on the task result

Reply:
Oh, thanks, added